css/at-rules/document.json

```diff
@@ -21,10 +21,27 @@
             "edge_mobile": {
               "version_added": null
             },
-            "firefox": {
-              "prefix": "-moz-",
-              "version_added": "1.5"
-            },
+            "firefox": [
+              {
+                "prefix": "-moz-",
+                "version_added": "1.5",
+                "version_removed": "61"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "61",
```

Comment:
Shouldn’t this, as a newer version, be first?

Or do we want to show the removed status first and the flag to enable it second?

Reply:
I agree, it makes sense to do this. I hadn't realised that the order of the JSON affected the order of the rendered output, but you're right, we should show the current status.